content/en/mobile/_index.md

```diff
@@ -22,7 +22,7 @@ You can log in using standard authentication, Google authentication, or [SAML][3
 
 #### Enabling SAML
 
-SAML login requires you to set up and authenticate your SAML provider with Datadog. For SAML IdP-initiated login, refer to the note at the end of this section. To authenticate SAML:
+SAML login requires you to set up and authenticate your SAML provider with Datadog. To authenticate SAML:
```

Comment:
The  flow for idp initiated login is still different then SP initiated SAML (entering the email to get the link wont give any valid link for IDP _only_ orgs)

Reply:
What do you suggest I change here?

Reply:
Keeping it as before?Was there an issue with the original sentence？

Reply:
I'll revert this change